feature-proposals.rst

```diff
@@ -13,34 +13,73 @@ community and decide if the community accepts or rejects the idea.
  Proposal Initiation
 *********************
 
-Proposal is formally initiated by creating an RFC on PHP wiki and announcing it
-on the list. If the proposal is a repeated discussion of an existing RFC, with
-or without modification, it still should be announced on the list for
-discussion.
-
-The announcement will be done in a way that's easy to flag & follow, e.g. - by
-``[RFC]`` in the subject line followed by the title of the RFC.
-
-The proposal should be initiated by one of its authors. If the proposal is a
-repeated one, re-proposed by somebody else, the proposer should discuss it with
-the original author, if possible, and add himself to the RFC author list before
-proposing it.
-
-If the proposer is not a member of php.net and thus can not create RFCs on the
-wiki, they should recruit one of the members for help or request membership.
+A proposal is formally initiated by creating an RFC on the PHP Wiki and
+announcing it on the ``internals@lists.php.net`` mailing list. If the proposal
+is a repeated discussion of an existing RFC, with or without modification, it
+MUST still be announced on the list for discussion.
+
+The official RFC discussion thread MUST use the prefix ``[RFC]`` followed by the
+title of the RFC as the Subject. The email MUST include a link to the Wiki page
+containing the RFC.
+
+The discussion thread MUST be initiated by one of the RFC's authors. If the
+proposal is a repeated one, re-proposed by somebody else, the proposer MUST
+discuss it with the original author and add themselves to the RFC author list
+before proposing it. If the original RFC authors do not agree or do not respond,
+a new Wiki page containing an RFC with a unique title MUST be created. In this
+case the original RFC's title is commonly suffixed with ``v2`` to indicate the
+relationship with the original RFC.
 
 *******************
  Discussion Period
 *******************
 
-There'd be a minimum of 2 weeks between when an RFC that touches the language is
-brought up on this list and when it's voted on is required. Other RFCs might use
-a smaller timeframe, but it should be at least a week. The effective date will
-not be when the RFC was published on the PHP wiki - but when it was announced on
-``internals@lists.php.net``, by the author, with the intention of voting on it.
-This period can be extended when circumstances warrant it - such as major
-conferences, key people being busy, force major events, or when discussion
-merits it - but should never be less than minimal time.
+Before an RFC may move to the voting phase a minimum discussion period of 2
+weeks (336 hours) MUST have elapsed. The discussion period starts at the time of
```

Comment:
Nitpicking here: Please do not assume 86,400-second days! 2 weeks should be sufficient here. If we start haggling about hours I think we have a bigger problem.

Reply:
After all:

```php
$now = new DateTimeImmutable();

var_dump($now->format('c'));
var_dump($now->add(new DateInterval('P2W'))->format('c'));
```

outputs

```
string(25) "2025-09-01T09:56:07+00:00"
string(25) "2025-09-15T09:56:07+00:00"
```

Which should be clear enough!

(https://3v4l.org/js9Xa)